src/biz/collab/ops.rs

```diff
@@ -521,6 +635,86 @@ pub async fn get_database_fields(
   Ok(acc)
 }
 
+// inserts a new field into the database
+// returns the id of the field created
+pub async fn add_database_field(
+  uid: i64,
+  collab_storage: &CollabAccessControlStorage,
+  pg_pool: &PgPool,
+  workspace_id: &str,
+  database_id: &str,
+  insert_field: AFInsertDatabaseField,
+) -> Result<String, AppError> {
+  let (mut db_collab, db_body) =
+    get_database_body(collab_storage, workspace_id, database_id).await?;
+
+  let new_id = uuid::Uuid::new_v4()
```

Comment:
why we need to do this? You can use nanoid id or just call `gen_field_id`

Reply:
I've used this